src/components/ThreejsEditorModal.js

```diff
@@ -108,6 +108,14 @@ export class ThreejsEditorModal extends ModalDialog {
         const loader = new THREE.ObjectLoader();
         const scene = loader.parse(materialsToThreeDSceneData(this.props.materials));
         this.editor.execute(new window.SetSceneCommand(scene));
+
+        // Add a light on the other side
```

Comment:
Let's move the code to `addDirectionalLight` function, please, and explain in the docstring why the addition is necessary